nats-base-client/protocol.ts

```diff
@@ -54,8 +54,13 @@ const FLUSH_THRESHOLD = 1024 * 32;
 
 export const INFO = /^INFO\s+([^\r\n]+)\r\n/i;
 
-export function createInbox(): string {
-  return `_INBOX.${nuid.next()}`;
+export function createInbox(inboxPrefix?: string): string {
+  if (typeof inboxPrefix === "string") {
+    inboxPrefix = inboxPrefix + ".";
+  } else {
+    inboxPrefix = "";
+  }
+  return `${inboxPrefix}_INBOX.${nuid.next()}`;
```

Comment:
I would make the default prefix be _INBOX prefix, which can then be replaced by the user-specified value.